flink-core/src/test/java/org/apache/flink/api/common/typeutils/CompositeTypeSerializerSnapshotTest.java

```diff
@@ -232,14 +232,11 @@ public void testRestoreCompositeTypeSerializer() throws IOException {
         // the restored nested serializer should be a RestoredNestedSerializer
         testSerializer = (TestCompositeTypeSerializer) testSerializerSnapshot.restoreSerializer();
         Assert.assertTrue(
-                testSerializer.getNestedSerializers()[0].getClass()
```

Comment:
That is wrong, isn't it? We should've restored the nested serializers as well. Why haven't we?

Reply:
Yes, it's updated incorrectly.
I have restored the code.